Lib/imaplib.py

```diff
@@ -135,10 +135,11 @@ class IMAP4:
 
     r"""IMAP4 client class.
 
-    Instantiate with: IMAP4([host[, port]])
+    Instantiate with: IMAP4([host[, port[, timeout]]])
```

Comment:
Again, specify the default as `None`. And in a following paragraph, the statement "All arguments are converted to strings" is no longer correct and needs to be fixed.